commands/reqlog.go

```diff
@@ -38,13 +38,13 @@ func (rl *ReqLog) AddEntry(rle *ReqLogEntry) {
 	rl.lock.Lock()
```

Comment:
We don't use `Signed-off-by`, you can omit it next time. :slightly_smiling_face: